00_Base/src/services/credentials.service.ts

```diff
@@ -410,7 +411,7 @@ export class CredentialsService {
     const serverVersion = serverVersionResponse[0];
     // TODO, should version url should be in DB?
     const serverVersionUrl =
-      'https://plugfest-dallas.demo.citrineos.app:445/ocpi/versions';
+      'https://plugfest.demo.citrineos.app:445/ocpi/versions';
```

Comment:
If you want, maybe we can temp add a query param for this serverVersionUrl besides this default value so that we are able to change it if necessary.

Reply:
@lydiazcheng if you are okay with it, perhaps we can do this as a fast follow? I can create a ticket for this on the board, only so that we can merge this in and have that incremental progress.. what do you think?

Reply:
That would be great. Thank you, @elliot-sabitov!